src/activities/morse_code/Morse_code.qml

```diff
@@ -0,0 +1,438 @@
+/* GCompris - MorseCode.qml
```

Comment:
rename the file MorseCode.qml (and change it in ActivityInfo.qml else it won't work)